source/_docs/blueprint/selectors.markdown

```diff
@@ -381,6 +382,38 @@ This selector does not have any other options; therefore, it only has its key.
 object:
 ```
 
+## Select selector
+
+The select selector shows a list of available options from which the user can choose. The value of the input contains the value of the selected option. Only a single option can be selected at a time.
+
+![Screenshot of a select selector](/images/blueprints/selector-select.png)
+
+The selector requires a list of options that the user can choose from.
+
+```yaml
+select:
+  options: []
```

Comment:
I don't think an empty list is a good thing to have in the documentation. Come to think of it, that should probably be disallowed in the backend code.

Reply:
Yeah, you're right. I thought `input_select` allowed to provide an empty list of options, but I checked and it's disallowed by the backend. I'll provide a fix in the backend PR and here.

Thank you for pointing it out. 👍🏻